Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkdir
+    (testpath/"test/.env").write "echo it works"
+    testcmd = "yes | bash -c '. #{opt_prefix}/activate.sh; autoenv_cd test'"
```

Comment:
Because a typical use of pipe_output concerns a test command that can handle any input, or at least a class of input. You specify one (usually simple) input and check the output. Here, there's only one input that makes any sense at all.